hedera-node/hedera-config/src/test/java/com/hedera/node/config/converter/LongPairConverterTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.config.converter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.hedera.node.config.types.LongPair;
+import com.swirlds.common.config.sources.SimpleConfigSource;
+import com.swirlds.config.api.Configuration;
+import com.swirlds.config.api.ConfigurationBuilder;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class LongPairConverterTest {
+
+    @Test
+    void testNullValue() {
+        // given
+        final LongPairConverter converter = new LongPairConverter();
+        final String value = null;
+
+        // then
+        assertThatThrownBy(() -> converter.convert(value)).isInstanceOf(NullPointerException.class);
+    }
+
+    @Test
+    void testValidValue() {
+        // given
+        final LongPairConverter converter = new LongPairConverter();
+        final String value = "1-2";
+
+        // when
+        final var result = converter.convert(value);
+
+        // then
+        assertThat(result.left()).isEqualTo(1L);
+        assertThat(result.right()).isEqualTo(2L);
+    }
+
+    @Test
+    void testInvalidValueMultipleSemicolon() {
```

Comment:
The name of this test is slightly misleading. 🙂

Reply:
fixed thanks